packages/landingpage-new/styles/globals.css

```diff
@@ -21,6 +22,10 @@ body {
   box-sizing: border-box;
 }
 
+p {
+  line-height: 25px;
+}
```

Comment:
Why is this necessary?

Reply:
Because it's not the default in the elements typography, even though it's specified in the design docs and used per default. Needs to be updated in the typography (body-large)